bridge/hardhat.config.js

```diff
@@ -52,7 +52,8 @@ module.exports = {
       ':SideToken',
       ':TransparentUpgradeableProxy$',
       ':NFTBridge$',
-      ':SideNFTToken'
+      ':SideNFTToken',
+      ':NFTERC721TestToken'
```

Comment:
I recomend using the abi IERC721

Reply:
never saw this, sorry! what I have available is `../../bridge/artifacts/contracts/zeppelin/token/ERC721/IERC721.sol/IERC721.json`, but throws an error when attempting to require that ABI (`Unhandled error: Error: You must provide the json interface of the contract when instantiating a contract object.`), any ideas?

Reply:
the thing, however, is that I need to mint a token, and only the test token contract allows this publicly

Reply:
never mine then, if you need to mint it, SideNFTToken is the way to go 👍